File: 1632.py
t = int(input())
for i in range(t):
    possibility = 1
    s = input()
    for j  in range(len(s)):
        c = s[j]
        if (c == 'A' or c == 'a' or
                c == 'E' or c == 'e' or
                c == 'I' or c == 'i' or
                c == 'O' or c == 'o' or
                c == 'S' or c == 's'):
            possibility *= 3
        else:
            possibility *= 2

    print(possibility)

File: 1632.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#define TAMMAX 50

int main() {
    int i, j;
    int t, possibility;
    char *s, c;

    scanf("%d", &t);
    s = (char *)malloc(TAMMAX * sizeof(char));
    for (i = 0; i < t; i++) {
        possibility = 1;
        scanf("%s", s);

        for(j = 0; j < strlen(s); j++) {
            c = s[j];
            if (c == 'A' || c == 'a' ||
                c == 'E' || c == 'e' ||
                c == 'I' || c == 'i' ||
                c == 'O' || c == 'o' ||
                c == 'S' || c == 's') {
                possibility *= 3;
                } else {
                    possibility *= 2;
                }
        }

        printf("%d\n", possibility);
    }

    return 0;
}
